### DISCUSSION TITLE
Easing.Evaluate in vanilla PrimeTween

### DISCUSSION THREAD

**Discussion by UrbiEtUrbi** [KyryloKuzyk/PrimeTween#100] [FAQ] [answered]
Would it be possible to implement Easing.Evaluate in PrimeTween without the need to add the DOTween adapter define?

**Comment by KyryloKuzyk** [comment, marked as answer]
This feature is currently experimental and requires additional scripting compilation define: https://discussions.unity.com/t/primetween-high-performance-animations-and-sequences/926420/215

I’ll think on adding this feature to the production version of PrimeTween. But can you please explain your use case for it?

**Comment by UrbiEtUrbi** [comment]
Hello, thanks for replying.

I use it in places where one would use Mathf.Lerp, but DOVirtual.EasedValue offers more customization.
For example in places when animating things outside of using tweens, like in the Update method or Coroutines. As well as a handy replacement for using an AnimationCurve.

**Comment by KyryloKuzyk** [comment]
I added the `Easing.Evaluate(float interpolationFactor, Ease ease)` method in version 1.1.22.
Huge thanks for using PrimeTween :)

**Reply by noio** [reply]
huge thanks for adding that! Saves me from adding another library to access these easing functions, 

my use case is like @UrbiEtUrbi's , when you want to do something from update, or sometimes it IS a tween, but i'm just setting multiple things:

```
Tween.Custom(  f=>SetAnimationProgress(f) )

void SetAnimationProgress(float progress){
  a = Easing.Evaluate(progress, Ease.OutBounce);
  b = Easing.Evaluate(progress, Ease.InQuad);
}
```

(yes i know i could just start 2 tweens! but then again sometimes i'm controlling this "animationProgress" from ANOTHER place without a tween, etc, etc)
